osu.Game/Overlays/Settings/SettingsTextBox.cs

```diff
@@ -8,7 +8,7 @@ namespace osu.Game.Overlays.Settings
 {
     public class SettingsTextBox : SettingsItem<string>
     {
-        protected override Drawable CreateControl() => new OsuTextBox
+        protected override Drawable CreateControl() => new OsuModSettingsTextBox
```

Comment:
`SettingTextBox` is also used in tournament client and change `OsuTextBox` to `OsuModSettingsTextBox` also affect and add border to it.

![Screenshot from 2021-06-12 10-20-45](https://user-images.githubusercontent.com/4964985/121763714-6cf58880-cb68-11eb-80c6-230e678e150c.png)

![Screenshot from 2021-06-12 10-23-19](https://user-images.githubusercontent.com/4964985/121763718-6ff07900-cb68-11eb-92b1-2e944e219ccd.png)

Reply:
I don't necessarily see this as a bad thing, but the naming is wrong. Also it would probably be a temporary change at best because the textbox already has a newer design which has not yet been implemented.

Reply:
Yeah, if the changes is intentionaly for general use. The component should be renamed.

Reply:
Renamed them to `OsuSettingsTextBox` and `OsuSettingsNumberBox`.